src/videojs-http-streaming.js

```diff
@@ -127,47 +127,64 @@ Vhs.canPlaySource = function() {
     'your player\'s techOrder.');
 };
 
-const emeKeySystems = (keySystemOptions, videoPlaylist, audioPlaylist) => {
+const emeKeySystems = (keySystemOptions, mainPlaylist, audioPlaylist) => {
   if (!keySystemOptions) {
     return keySystemOptions;
   }
 
-  const codecs = {
-    video: videoPlaylist && videoPlaylist.attributes && videoPlaylist.attributes.CODECS,
-    audio: audioPlaylist && audioPlaylist.attributes && audioPlaylist.attributes.CODECS
-  };
+  const codecs = {};
 
-  if (!codecs.audio && codecs.video && codecs.video.split(',').length > 1) {
-    codecs.video.split(',').forEach(function(codec) {
-      codec = codec.trim();
+  if (mainPlaylist && mainPlaylist.attributes && mainPlaylist.attributes.CODECS) {
+    const mainCodecs = mainPlaylist.attributes.CODECS;
```

Comment:
perhaps we should use `parseCodecs` from vhs utils and loop through that. Should we make sure that we can support the codec here as well?

Reply:
Will switch to using `parseCodecs`. On support, since this should be called on `createdsourcebuffers`, I think we should be safe.